posts/multiple_index_xarray_xoak.md

```diff
@@ -0,0 +1,56 @@
+---
+author: Max Grover
+date: 2021-4-23
+tags: xarray, xoak, index
+---
+
+# Dealing with Multi-Indexing Coordinates and the Power of Xoak
+
+This week, there a post within the Zulip regarding how to deal with indexing [CAM-SE](https://agupubs.onlinelibrary.wiley.com/doi/full/10.1029/2017MS001257) data. The tricky part here is that is an unstructured grid, where there is only one column to the data, `ncol`.
+
+Using the [Multiple-level indexing documentation](http://xarray.pydata.org/en/stable/indexing.html#multi-level-indexing) within Xarray didn't seem to help... so what other options are there?
```

Comment:
Let's add the repr from `/glade/work/yeager/iHesp/fredc_data/B1850/B.E.13.B1850C5.ne120_t12.sehires38.003.sunway_02.cam.h.PSL.002101.050112.nc` here. You can use `xr.set_options(display_style="text")` to get a text version.